projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * WordPress.com admin bar
+ *
+ * Modifies the WordPress admin bar with WordPress.com-specific stuff.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+use Automattic\Jetpack\Jetpack_Mu_Wpcom;
+
+if ( get_option( 'wpcom_admin_interface' ) !== 'wp-admin' ) {
+	return;
+}
+
+/**
+ * Enqueue assets needed by the WordPress.com admin bar.
+ */
+function wpcom_enqueue_admin_bar_assets() {
+	wp_enqueue_style(
+		'wpcom-admin-bar',
+		plugins_url( 'build/wpcom-admin-bar/wpcom-admin-bar.css', Jetpack_Mu_Wpcom::BASE_FILE ),
+		array(),
+		Jetpack_Mu_Wpcom::PACKAGE_VERSION
+	);
+}
+add_action( 'admin_bar_menu', 'wpcom_enqueue_admin_bar_assets' );
+
+/**
+ * Repurposes the WP logo as a link to /sites.
+ *
+ * @param WP_Admin_Bar $wp_admin_bar The WP_Admin_Bar core object.
+ */
+function wpcom_repurpose_wp_logo_as_all_sites_menu( $wp_admin_bar ) {
+	if ( is_agency_managed_site() ) {
+		return;
+	}
+
+	foreach ( $wp_admin_bar->get_nodes() as $node ) {
+		if ( $node->parent === 'wp-logo' || $node->parent === 'wp-logo-external' ) {
+			$wp_admin_bar->remove_node( $node->id );
+		}
+	}
+
+	$wp_admin_bar->remove_node( 'wp-logo' );
+	$wp_admin_bar->add_node(
+		array(
+			'id'    => 'wp-logo',
+			'title' => '<span class="ab-icon" aria-hidden="true"></span><span class="screen-reader-text">' .
+						/* translators: Hidden accessibility text. */
+						__( 'All Sites', 'jetpack-mu-wpcom' ) .
+						'</span>',
+			'href'  => 'https://wordpress.com/sites',
+			'meta'  => array(
+				'menu_title' => __( 'All Sites', 'jetpack-mu-wpcom' ),
+			),
+		)
+	);
+}
+add_action( 'admin_bar_menu', 'wpcom_repurpose_wp_logo_as_all_sites_menu', 11 );
+
+/**
+ * Adds the Reader menu.
+ *
+ * @param WP_Admin_Bar $wp_admin_bar The WP_Admin_Bar core object.
+ */
+function wpcom_add_reader_menu( $wp_admin_bar ) {
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

Should we move the Reader menu to the right most according to the design in pfsHM7-1kz-p2 or https://github.com/Automattic/dotcom-forge/issues/8144? Did I miss something? 

this PR
<img width="346" alt="Screenshot 2024-07-09 at 11 09 03" src="https://github.com/Automattic/jetpack/assets/5287479/4fa43828-d623-49f0-80de-107e071da2a4">

Figma
<img width="293" alt="Screenshot 2024-07-09 at 11 10 37" src="https://github.com/Automattic/jetpack/assets/5287479/95fad2e4-89d0-439f-a757-752bc4f56abc">

Reply:
Good question! We had a p1720413874823569/1720204941.135799-slack-C06DN6QQVAQ about this on Slack, and @davemart-in approved this:

> > Would it be worth putting Reader on the left like this? It's a bit more global than the site level and means it won't move around when navigating in and out of site contexts.
> 
> Yes. Let's do that!